### DISCUSSION TITLE
关于输入框增强, 添加润色功能.

### DISCUSSION THREAD

**Discussion by XMB-7** [immersive-translate/immersive-translate#1391] [Ideas]
### 新功能 | New Feature

1. 假设是汉译英, 目前输入汉语, 可以一键翻译成英语, 但如果输入是英语, 那么调用相同功能就是没有响应的. 建议此时可以添加润色功能, 即输入汉语时, 输入框增强功能表现为翻译, 而输入英语时, 输入框增强功能表现为润色.
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1. 翻译和润色, 是国人使用英语时两大最常用的应用场景. 一个是输入, 一个是输出. 
2. 输入框增强这个功能很实用, 那么为什么不把润色也囊括进来呢.
3.

### 提议 | Proposal

添加另一套用于润色的提示词.
或者干脆把翻译和润色就分成两个操作, 从输入框增强里剥离出来.

**Comment by theowenyoung** [comment]
不错的idea!

(但是似乎会让功能再度膨胀.. 所以短期内可能还是不会新增这个特性.
